File: OpenLab.Kitchen.Receiver.RFID/FeigRFID/Samples/VB/MultipleUSBReader/VS2010/Connect.vb
Imports OBID
Imports System


Public Class Connect
    Implements FeUsbListener
    Public running As MultipleUSBReader

    Public Sub OnConnectReader(ByVal DeviceHandle As Integer, ByVal DeviceID As Long) Implements OBID.FeUsbListener.OnConnectReader
        running.AddDevice(DeviceID)
    End Sub

    Public Sub OnDisConnectReader(ByVal DeviceHandle As Integer, ByVal DeviceID As Long) Implements OBID.FeUsbListener.OnDisConnectReader
        running.RemoveDevice(DeviceID)
    End Sub
End Class

File: OpenLab.Kitchen.Receiver.RFID/FeigRFID/Samples/C#/MultipleUSBReader/VS2010/Connect.cs
using System;
using System.Collections.Generic;
using System.Text;
using OBID;

namespace MultipleUSBReader
{
  
    class Connect : FeUsbListener
        {
            public MultipleUSBReader running;
           
            
            public  void OnConnectReader(int DeviceHdl, long DeviceID)
                {
                    running.AddDevice(DeviceID);
                }
            public  void OnDisConnectReader(int DeviceHdl, long DeviceID)
                {
                    running.RemoveDevice(DeviceID);
                }
        }
   
}
